content/en/docs/tasks/configure-pod-container/static-pod.md

```diff
@@ -38,6 +38,10 @@ The `spec` of a static Pod cannot refer to other API objects
 {{< glossary_tooltip text="Secret" term_id="secret" >}}, etc).
 {{< /note >}}
 
+{{< note >}}
+Static pods do not support [ephemeral containers](/docs/concepts/workloads/pods/ephemeral-containers/).
```

Comment:
I'd use a glossary tooltip here (not a plain Markdown hyperlink).

---

This is a lot of notes now.
For this file, consider turning one of the other note callouts into a plain paragraph,. For example, the restriction on referring to other API objects doesn't really need to be a note.

Even better, how about add a heading "Limitations on static pods" and list each limitation under that heading. This could come later in the page with a single note callout earlier on that links to the specific section about limitations.